client/js/views/file.react.jsx

```diff
@@ -9,117 +9,69 @@ var jsDiff = require('diff');
 var File = React.createClass({
   getInitialState: function() {
     return {
-      html : ''
+      diff : ''
```

Comment:
I redid this because it was rendering twice maybe due to having xml/view elements in the state, which should only be data/model